packages/date-picker/src/DatePicker.tsx

```diff
@@ -19,38 +19,34 @@ export const DatePickerWrapper: React.FunctionComponent<DatePickerProps> = ({
   onDayChange,
   labelText,
   isDisabled = false,
-}) => (
-  <>
-    {labelText && <Label disabled={isDisabled} labelText={labelText} />}
-    <div
-      className={classnames(
-        styles.wrapper,
-        styles.withStartIconAdornment,
-        styles.withSearch,
-        {
-          [styles.withDisabled]: isDisabled,
+}) => {
+  const getNavbar = ({ ...navbarProps }: CalendarNavProps) => (
```

Comment:
tbh I'd just inline that on line 46, makes it easier see what it is without having to scan the file up/down:
```
...
getNavbar={({ ...navbarProps }: CalendarNavProps) => <CalendarNav {...navbarProps} />}
...
```